src/pages/best-practices.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Best Practices for PWA Studio
+---
+
+# Best Practices for PWA Studio
+
+Here are some best practices to keep in mind when working with PWA Studio.
+
+## Dynamic Content
+
+Using dynamic content can be challenging due to shifting layouts. It is helpful to end users if the page layout is established as quickly as possible. Dynamic content can lead to page flow issues.
+
+- Use sizing as often as possible. If blocks have defined heights and widths, the layout engine can reserve those spaces faster. Use shims to enable block out image areas.
```

Comment:
Is this supposed to say _shimmers_ instead of _shims_?

Our `Shimmer` component reserves spaces on the page.